checks/process.go

```diff
@@ -63,10 +63,10 @@ func (p *ProcessCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Mess
 		return nil, err
 	}
 
-	containers, err := container.GetContainers()
-	if err != nil && err != docker.ErrDockerNotAvailable {
+	containers, errs := container.GetContainers()
+	if len(errs) != 0 {
 		containers = []*docker.Container{}
-		log.Warn("Omitting container info for process check due to error: %s", err)
+		log.Warn("omitting container info for process check due to retrieval errors")
```

Comment:
is this noisy and redundant with the errors we are logging in `GetContainers()`?